Code/Editor/CryEdit.cpp

```diff
@@ -2299,9 +2299,30 @@ int CCryEditApp::IdleProcessing(bool bBackgroundUpdate)
             GetIEditor()->Notify(eNotify_OnIdleUpdate);
         }
     }
-    else if (GetIEditor()->GetSystem() && GetIEditor()->GetSystem()->GetILog())
+    else
     {
-        GetIEditor()->GetSystem()->GetILog()->Update(); // print messages from other threads
+        if (GetIEditor()->GetSystem() && GetIEditor()->GetSystem()->GetILog())
+        {
+            GetIEditor()->GetSystem()->GetILog()->Update(); // print messages from other threads
+        }
+
+        // If we're backgrounded and not fully background updating, idle to rate limit SystemTick
+        static AZ::TimeMs sTimeLastMs = AZ::GetRealElapsedTimeMs();
+
+        const auto console = AZ::Interface<AZ::IConsole>::Get();
+        AZ::TimeMs maxFrameTimeMs = AZ::TimeMs{ 1 };
+        console->GetCvarValue("ed_backgroundSystemTickCap", maxFrameTimeMs);
```

Comment:
I can't look at it now but if you declare i think there is an AZ_CVAR_EXTEND where you define the cvar in the header and deceleration in the cpp. you just need to include the setting and it should let you just reference the property instead of querying it from the console.

Reply:
seems a lot less error prone since your are relying on a string.